aspnetcore/includes/bind7.md

```diff
@@ -13,7 +13,7 @@ Create the following `PositionOptions` class:
 
 An options class:
```

Comment:
It might be worth adding 'An options class, when used as the generic parameter for the `Get<>` method, as `Bind` takes anything, including a concretion of an abstract class, e.g. given:

```
public abstract class SomethingWithAName
{
    public abstract string Name { get; set; }
}

public class PositionOptions : SomethingWithAName
{
    public PositionOptions(int age)
    {
        Age = age;
    }

    public const string Position = "Position";

    public override string Name { get; set; }
    public string Title { get; set; } = String.Empty;

    public int Age { get; set; }
}
```

... you can call:

```
SomethingWithAName p = new PositionOptions(42);
app.Configuration.Get("Position", p);
```